scripts/criu-ns

```diff
@@ -80,8 +88,21 @@ def run_criu(args):
     Spawn CRIU binary
     """
     print(sys.argv)
-    os.execlp('criu', *['criu'] + args)
-    raise OSError(errno.ENOENT, "No such command")
+
+    if "--criu-binary" in args:
+        try:
+            opt_index = args.index("--criu-binary")
+            path = args[opt_index + 1]
+            args.pop(opt_index)
+            args.pop(opt_index)
+            args.insert(0, "criu")
+            os.execv(path, args)
+            raise OSError(errno.ENOENT, "No such command")
+        except (ValueError, IndexError, FileNotFoundError):
+            raise OSError(errno.ENOENT, "--criu-binary missing argument")
+    else:
+        os.execlp("criu", *["criu"] + args)
```

Comment:
Can we do `exec` calls symmetrically? E.g. maybe we can:
```suggestion
        args.insert(0, "criu")
        os.execvp("criu", args)
```
I mean using execl and execv in one piece of code may be confusing. Let's chose either we have vector arguments everywhere or we have variable number of function arguments everywhere. I prefer the former.

Reply:
Yeah, I agree. Done!